File: basic/Ex02/codigo.py
def peso_no_planeta():
    print("===  PLANETAS  ===")
    print("\n1. Mercúrio - 0,37\n2. Vênus - 0,88\n3. Marte - 0,38\n4. Júpiter - 2,64\n5. Saturno - 1,15\n6. Urano - 1,17\n")

    planeta = int(input("Informe o número do planeta desejado: "))
    peso = float(input("Informe seu peso na Terra: "))

    match planeta:
        case 1:
            pesoPlaneta = (peso / 10) * 0.37
        case 2:
            pesoPlaneta = (peso / 10) * 0.88
        case 3:
            pesoPlaneta = (peso / 10) * 0.38
        case 4:
            pesoPlaneta = (peso / 10) * 2.64
        case 5:
            pesoPlaneta = (peso / 10) * 1.15
        case 6:
            pesoPlaneta = (peso / 10) * 1.17
        case _:
            print("Opção inválida.")
            return

    print(f"Seu peso no planeta desejado é: {pesoPlaneta:.2f}")



peso_no_planeta()

File: basic/Ex02/codigo.c
/******************************************************************************

2. Escrever um algoritmo que
obtenha o peso de uma pessoa na Terra e o
número de um planeta. Ao final, com auxílio da
tabela abaixo, calcular o peso desta pessoa no
planeta escolhido.

1 Mercúrio 0,37
2 Vênus 0,88
3 Marte 0,38
4 Júpiter 2,64
5 Saturno 1,15
6 Urano 1,17

Para calcular o peso no planeta escolhido,
utilize a seguinte fórmula:
pesoPlaneta = (pesoTerra/10) * gravidadePlaneta

*******************************************************************************/

#include <stdio.h>

int main()
{
    float pesoPlaneta, pesoTerra, gravidade;
    int planeta;                                //Definindo as váriaveis 
    
    printf("Informe seu peso na Terra: ");
    scanf("%f", &pesoTerra);
    
    printf("Informe o número do planeta desejado: ");
    printf("\n 1. Mercúrio - 0,37 \n 2. Venus - 0,88\n 3. Marte - 0,38\n 4. Jupiter - 2,64\n 5. Saturno - 1,15\n 6. Urano - 1,17\n");
    scanf("%d", &planeta);
    
   switch(planeta) {
       
       case 1:
            gravidade = 0.37;
            break;
        case 2:
            gravidade = 0.88;
            break;
        case 3:
            gravidade = 0.38;
            break;
        case 4:
            gravidade = 2.64;
            break;
        case 5:
            gravidade = 1.15;
            break;
        case 6:
            gravidade = 1.17;
            break;
   }
   
   pesoPlaneta = (pesoTerra/10) * gravidade;
   
   printf("Seu peso no planeta escolhido é: %.2f.", pesoPlaneta);

    

    return 0;
}

